server/routes/printReport/NoticeOfBeingPutOnReportData.ts

```diff
@@ -0,0 +1,36 @@
+import { ConfirmedOnReportData } from '../../data/ConfirmedOnReportData'
```

Comment:
Not exciting, but given that this is just a class, maybe better to put in the `data` directory.

We can do that later though